magda-indexer/src/main/scala/au/csiro/data61/magda/indexer/external/registry/RegisterWebhook.scala

```diff
@@ -58,18 +61,20 @@ object RegisterWebhook {
         EventType.DeleteRecord,
         EventType.CreateRecordAspect,
         EventType.PatchRecord,
-        EventType.DeleteRecordAspect),
+        EventType.DeleteRecordAspect,
+        EventType.DeleteRecord),
```

Comment:
`DeleteRecord` already exists, third from the top.

Reply:
Good point...